### ISSUE TITLE
Update coordinates

### ISSUE THREAD

**Issue by geaplanet** [jmoenig/Snap--Build-Your-Own-Blocks#695]
There is one detail in Scratch that it is useful to do sprites animations. Blocks "go to x:_ y:_" and "glide _ secs to x:_ y:_"  update coordinates when a sprite is moved in the stage.

**Comment by studej** [comment]
Do you talk about <i>x</i> and <i>y</i> coordinates watchers? Their values are updated, but slowlier than in Scratch

**Comment by bromagosa** [comment]
Nope, I think he means that the block template gets the current Sprite coordinates updated automatically, but this is quite hard to achieve in Snap!...

**Comment by studej** [comment]
Aha, I understand now.
I must say that sometimes I like as it is in Scratch and sometimes as it is in Snap :)
